sync-core/src/test/java/com/cloudant/sync/util/TestUtils.java

```diff
@@ -153,5 +148,27 @@ public static File loadFixture(String fileName){
        }
    }
 
+   public static SQLDatabase getDatabaseConnectionToExsitingDb(SQLDatabase db){
+       if(Misc.isRunningOnAndroid())
+           return db;
+
+       try {
+           String filePath = (String) db.getClass()
```

Comment:
At the time of writing, I didn't want to make a test dependancy on JavaSE, since during the android builds java-se is not compiled, only core and android are